CHANGELOG.md

```diff
@@ -14,8 +14,10 @@
   - UI updates of order status button and back button
 - Service Release Process
   - Updated technicalUserProfile by allowing 'none' for dataspace services
+  - Support optional terms for contract and consent
 - App Release Process
   - Updated technicalUserProfile by allowing 'none'
+  - Support optional terms for contract and consent
```

Comment:
is this comment relevant?

Reply:
@manojava-gk Not added by me. May be get added by mistake. Removed them now.